icees_api/handlers.py

```diff
@@ -210,6 +210,8 @@ def feature_association(
             feature_a,
             feature_b,
         )
+        if not return_value['feature_matrix']:
```

Comment:
See my comment [here](https://github.com/NCATS-Tangerine/icees-api/issues/217#issuecomment-1022585412)

Reply:
@maximusunc I agree with everything you mentioned in that comment. I think what the example @karafecho gave in that issue may very likely come down to an empty association matrix, which explains all those null in the response. That is why I think the fix in this PR most likely will fix both issues since they appear to have the same root cause. I'd suggest to get this PR out of the door first, and create a new issue for input validation as needed.

Reply:
Could you explain how this PR fixes both the issues mentioned please?

Reply:
@maximusunc This PR fixes the issue #212 for sure since I tested it locally. I am not sure whether it fixes the issue #217, but by looking at the queries @karafecho made in the screen captures, I see the response looks like the response I got when I entered a year for which there is no data for DILI instance which I tested locally for feature associations endpoint. So I speculated the root cause for the issue #217 might be the same as the issue #212, i.e., there were empty feature associations for the example queries. It is just a speculation, though, since I did not test it, but after this PR is merged, I'll deploy it to asthma and DILI instances and I can test the example queries to see if this PR will indeed fix the issue #217. If not, I can look into it later.